Makefile

```diff
@@ -238,7 +238,7 @@ CIRRUS_ENABLE = export CIRRUS=1 &&
 CIRRUS_BLACK_CHECK = black -l 90 --check --diff .
 CIRRUS_BLACK_FIX = black -l 90 .
 CIRRUS_RUFF_CHECK = ruff check .
-CIRRUS_RUFF_FIX = ruff --fix .
+CIRRUS_RUFF_FIX = ruff check --fix .
```

Comment:
Oh dang I forgot to update this for Cirrus too XDDD whoops sorry

Reply:
no worries :)